torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3384,16 +3391,70 @@ at::Tensor interpolate(
     }
   }
 
+  double scale_factors_1 = -1.0f;
```

Comment:
nit: `-1.0f` -> `1.0`

Reply:
good catch. I will wait for the CI to finish running to confirm that there are no new failures, and will then make this fix.